TUIFIManager/TUIFIProfile.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, text, color_map=1, open_with=None):
     ),2, 'vim'),
     '.js':TUIFIProfile(( 
         '  ▃▃▃ ▃▃▃ \n'
-        'JS▔█▔ █▃▃ \n'  
```

Comment:
Personally i kind of like the 'JS'  letters at the left side, they give a Z-Axis depth, as if they were at the background and projected to the foreground